frontend/apps/marketing/src/components/list/List.tsx

```diff
@@ -0,0 +1,52 @@
+import {Entry, EntryFields} from 'contentful';
+import List, {
+  DEFAULT_ICON,
+  ListProps,
+  ListItem,
+} from '@code-dot-org/component-library/list';
+import {useMemo} from 'react';
+
+type ListItemEntry = Entry & {
+  fields: {
+    shortText: EntryFields.Text;
+  };
+};
+
+interface ListContentfulProps extends Omit<ListProps, 'items'> {
+  items?: ListItemEntry[];
+  iconName?: string;
+}
+
+const ListContentful: React.FunctionComponent<ListContentfulProps> = ({
+  items = [],
+  iconName = DEFAULT_ICON,
+  ...props
+}) => {
+  const listItems: ListItem[] = useMemo(
+    () =>
+      items.filter(Boolean).map(listItemEntry => ({
+        label: listItemEntry.fields.shortText,
+      })),
+    [items],
+  );
+
+  // Show placeholder text until a content entry is added
+  if (!listItems.length) {
+    return (
+      <em>
+        <strong>✍ List placeholder.</strong> Please add a "List" content type
+        entry in the List sidebar.
+      </em>
+    );
+  }
+
+  return (
+    <List
+      {...props}
+      items={listItems}
+      icon={{iconName: iconName || DEFAULT_ICON}}
```

Comment:
We also need to support brand icons here. We already have it in the code base, you can see it [here](https://github.com/code-dot-org/code-dot-org/blob/3461bab9b3a247e282bad7b328c192195b9b31a6/frontend/apps/marketing/src/components/button/Button.tsx#L34-L36)


@moshebaricdo please confirm that we won't need our custom icons here as well

Reply:
also nit: since iconName already has two layers of default values

```suggestion
      icon={{iconName, iconFamily: isBrandIcon ? 'brands' : undefined,}}
```

Reply:
No need for custom icons, yes to brand icons.